### PR TITLE
[LOGGING] introduce LoggingSupport

### PR DESCRIPTION
**Pull request by fgdrf**
to avoid duplicated code overall Plug-Ins and unified bahavior

Change-Id: Ida157f1ef58e24543465e59a4e7de8033de63d96
Signed-off-by: Frank Gasdorf <fgdrf@users.sourceforge.net>

### CODE REVIEW

**Comment by fgdrf** [comment]
@olsmn  & @sschulz92 may you have a look at?

File: docs/devel/en/coding_conventions.rst [hunk 1]
```diff
@@ -32,6 +32,23 @@ There is a utility method in URLUtils that will do this for you:
 
 Use it!
 
+Logging in Bundles
+------------------
+To write logs that appears in the Error Log View with different states, its recommend to use LoggingSupport-class of core bundle.
```
Comment (olsmn) [inline docs/devel/en/coding_conventions.rst:37]: Only little things:
... logs that appear ... it is recommended ...

Comment (fgdrf) [reply docs/devel/en/coding_conventions.rst]: Thank you for the feedback! Documentation is fixed

**Review by sschulz92 — APPROVED** [review]
👍

**Comment by fgdrf** [comment]
Thank you for your feedback and review!

**Pull request merged**